### DISCUSSION TITLE
JVP and VJP for tensor-valued function

### DISCUSSION THREAD

**Discussion by exenGT** [jax-ml/jax#9930] [Q&A] [answered]
Hi,

How can I calculate the JVP and VJP for tensor-valued function, like the following:
![image](https://user-images.githubusercontent.com/13700125/158777662-0058ffe4-5bbc-429e-8090-0ee343252ae4.png)
Here, A is a tensor, x can be a vector or a tensor.

Thanks,
JW

**Comment by YouJiacheng** [comment, marked as answer]
`jax.jvp` and `jax.vjp` is compatible with tesnor-valued functions. No extra effort is needed.
